core/src/bin/run.ts

```diff
@@ -34,6 +34,14 @@ export class RunCLI extends CLI {
       letter: "w",
       flag: true,
     },
+    scheduleIds: {
+      description: "Only run specific Schedules by id",
+      required: false,
+      variadic: true,
+      letter: "s",
+      placeholder: "schedule ids",
+      formatter: (v: any) => v as boolean | string[],
```

Comment:
This is the case when `variadic: true`... I wonder if there's a way to do this automatically

Reply:
probably not :/

Not that we need a test for it explicitly, but can you confirm that using the CLI always returns an array, even if you provide just 1 scheduleId?  I'm a little worried about `grouparoo run -s abc123` coming in as a string here

Reply:
Yeah, it does:

```
▲ staging-enterprise % roo run -s foo
notice: Modified your runtime environment with /Users/pedro/grouparoo/grouparoo/apps/staging-enterprise/.env 
warning: Plugin @grouparoo/sentry requires environment variable SENTRY_DSN to be set 
warning: Plugin @grouparoo/sentry requires environment variable SENTRY_TRACE_SAMPLE_RATE to be set 
scheduleIds: [ 'foo' ]
```

```
▲ staging-enterprise % roo run -s foo bar
notice: Modified your runtime environment with /Users/pedro/grouparoo/grouparoo/apps/staging-enterprise/.env 
warning: Plugin @grouparoo/sentry requires environment variable SENTRY_DSN to be set 
warning: Plugin @grouparoo/sentry requires environment variable SENTRY_TRACE_SAMPLE_RATE to be set 
scheduleIds: [ 'foo', 'bar' ]
```

When adding the param I wanted to add a test for this but encountered a `// NOTE: The command cannot be run in full as the config/env is already set at boot` 🤷🏽‍♂️